samples/matter/thermostat/src/temperature_measurement/real_temp.cpp

```diff
@@ -23,15 +31,44 @@ constexpr chip::EndpointId mTemperatureMeasurementEndpointId = 1;
 
 RealSensor::RealSensor()
 {
-	BindingHandler::GetInstance().Init();
+	BindingHandler::Init();
 }
 
 void RealSensor::TemperatureMeasurement()
 {
 	BindingHandler::BindingData *data = Platform::New<BindingHandler::BindingData>();
-	data->clusterId = Clusters::TemperatureMeasurement::Id;
-	data->localEndpointId = mTemperatureMeasurementEndpointId;
-	data->isGroup = BindingHandler::GetInstance().IsGroupBound();
-	DeviceLayer::PlatformMgr().ScheduleWork(BindingHandler::ThermostatWorkerHandler,
-						reinterpret_cast<intptr_t>(data));
+	data->ClusterId = Clusters::TemperatureMeasurement::Id;
+	data->EndpointId = mTemperatureMeasurementEndpointId;
+	data->IsGroup = BindingHandler::IsGroupBound();
```

Comment:
We are reading this flag from BindingHandler and then passing into the BindingHandler::RunBoundClusterAction... so it looks to me like something that can be handled solely by the BindingHandler with no need for additional action in the application. Or do I miss something here?

Reply:
Asked Patryk about it as well but we thought we could change that in the followup. Long story short - shell commands use this force unicast/groupcast binding (I don't think that's very useful tbh and I'm +1 for making this transparent for a user).

Reply:
Ok, I'm ok with follow ups as long as we we create a task in the according epic right away :)

Reply:
Approving on condition that we create aforementioned task.